modules/unpickler/src/main/scala/ch/epfl/scala/debugadapter/internal/stacktrace/Scala3Unpickler.scala

```diff
@@ -57,17 +59,42 @@ class Scala3Unpickler(
       case None =>
         throw new Exception(s"Cannot find Scala symbol of $fqcn")
       case Some(declaringType) =>
-        val matchingSymbols =
-          declaringType.declarations
-            .collect { case sym: TermSymbol if sym.isTerm => sym }
-            .filter(matchSymbol(method, _))
-
-        if matchingSymbols.size > 1 then
-          val message =
-            s"Found ${matchingSymbols.size} matching symbols for $method:" +
-              matchingSymbols.mkString("\n")
-          throw new Exception(message)
-        else matchingSymbols.headOption
+         matchesLocalMethodOrLazyVal(method) match
+          case Some((methodName, index)) => findLocalMethodOrLazyVal(declaringType,methodName,index)
+          case _ => 
+              val matchingSymbols = declaringType.declarations
+              .collect { case sym: TermSymbol if sym.isTerm => sym }
+              .filter(matchSymbol(method, _))
+               if matchingSymbols.size > 1 then
+               val message =s"Found ${matchingSymbols.size} matching symbols for $method:" +
+               matchingSymbols.mkString("\n")
+               throw new Exception(message)
+               else matchingSymbols.headOption
```

Comment:
In the CI, the `Check Formatting` failed with a `Parse Error`:

```
org.scalafmt.dynamic.exceptions.PositionExceptionImpl: /home/runner/work/scala-debug-adapter/scala-debug-adapter/modules/unpickler/src/main/scala/ch/epfl/scala/debugadapter/internal/stacktrace/Scala3Unpickler.scala:72: error: [dialect scala3] ; expected but else found
               else matchingSymbols.headOption
               ^: /home/runner/work/scala-debug-adapter/scala-debug-adapter/modules/unpickler/src/main/scala/ch/epfl/scala/debugadapter/internal/stacktrace/Scala3Unpickler.scala
```

Could you indent the `if else` correctly and then run `scalafmt` to format everything else?

```suggestion
              val matchingSymbols = declaringType.declarations
                .collect { case sym: TermSymbol if sym.isTerm => sym }
                .filter(matchSymbol(method, _))
               if matchingSymbols.size > 1 then
                 val message =s"Found ${matchingSymbols.size} matching symbols for $method:" +
                 matchingSymbols.mkString("\n")
                 throw new Exception(message)
               else matchingSymbols.headOption
```